apps/emqx/src/emqx_channel.erl

```diff
@@ -1398,19 +1411,16 @@ terminate(_, #channel{conn_state = idle} = _Channel) ->
     ok;
 terminate(normal, Channel) ->
     run_terminate_hook(normal, Channel);
-terminate({shutdown, kicked}, Channel) ->
-    run_terminate_hook(kicked, Channel);
 terminate({shutdown, Reason}, Channel) when
-    Reason =:= discarded;
-    Reason =:= takenover
+    Reason =:= expired orelse
```

Comment:
Reason =:= expired orelse
Reason =:= takenover orelse
Reason =:= kicked orelse
Reason =:= discarded

Reply:
they are same as my changes?

code formating is done by `make fmt`, not me.